.github/config_files/config_lint.yaml

```diff
@@ -0,0 +1,11 @@
+checks:
```

Comment:
Is this a requirement from the github action?

Reply:
yes